### PR TITLE
added card widget

### PR DESCRIPTION
**Pull request by sibteali786**
## Describe your changes
Added Card Widget
### Screenshots [Optional]
image

## Issue ticket number and link
#65 
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] I have added tests

### CODE REVIEW

File: apps/starter/src/ensemble/home.yaml [hunk 1]
```diff
@@ -193,8 +193,43 @@ View:
...
+                          name: user
+                          template:
+                            Card:
+                              styles:
+                                display: flex
+                                flexDirection: column
+                                justifyContent: space-between
+                                alignItems: flex-start
+                                gap: 16
+                                padding: 24
```
Comment (evshi) [inline apps/starter/src/ensemble/home.yaml:212]: Can we just a Column widget inside instead of requiring this styling?

**Comment by sibteali786** [comment]
@evshi i have added a very basic card widget for now, its yaml is similar to what Sana did used for CardComponent except Image property. This is what we can achieve with it. Although i am not sure about current yaml as it is more relevant to web than flutter. What do you suggest  ? 

Like for instance -> instead of crossAxis we are using justifyItems here

File: packages/runtime/src/widgets/Card.tsx [hunk 2]
```diff
@@ -1,37 +1,67 @@
...
+      | "space-between"; //justify options
+    gap?: number;
+    border: string;
+    borderRadius: string;
+    borderWidth: string;
+    boxShadow: string;
+    width: number;
+    height: number;
+    margin: string;
+    padding: string;
```
Comment (evshi) [inline packages/runtime/src/widgets/Card.tsx:38]: See my comment above but let's not expose all these properties on the card widget. The purpose of the card widget is to provide the card styling, not to control the internal layout. Users can use other widgets like Row and Column to control internal layout.

Some styles that we _can_ expose for Card would be for:

shadow
border
border-radius
padding

Comment (sibteali786) [reply packages/runtime/src/widgets/Card.tsx:38]: Okay, I get it.  One question though, let us suppose if a user wants more gap in between items of card ? should we allow it or not. How can we control it if we want to for some reason ?

Comment (sibteali786) [reply packages/runtime/src/widgets/Card.tsx:38]: Okay i get it, let me update it

**Review by evshi — APPROVED** [review]
Looks good! Please rebase the latest changes for the home yaml

**Pull request merged**